io.github.narunlifescience.AlphaPlot.yml

```diff
@@ -0,0 +1,78 @@
+app-id: io.github.narunlifescience.AlphaPlot
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+command: alphaplot
+rename-icon: alphaplot
+rename-desktop-file: alphaplot.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --env=SESSION_MANAGER= # required to avoid Qt warning
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
```

Comment:
Any reason for those? Not sure if the app uses QFileDialog but that uses the portal.

Reply:
The app uses QFileDialog ([reference](https://github.com/narunlifescience/AlphaPlot/blob/70c42625fbe3a3ca81875b27d1ca93efb2955d9d/alphaplot/src/ui/ExtensibleFileDialog.h#L32)).

This is the difference between "Save" and "Load" a project.

![AlphaPlot-save-open](https://user-images.githubusercontent.com/50991168/94759056-421a2a00-0375-11eb-865d-34294b8d4a0b.jpg)

Enable "filesystem=home"? What do you suggest?

Reply:
What I meant is that if the app uses QFileDialog, it uses the portal by default which means you don't need filesystem permissions to open/save files.

Reply:
The image shows that **save** a project does not really require file system permissions, that's correct. However, to **open** a project, it needs filesystem permissions. In **File > Export Graph** opens a similar File Dialog, in which the app needs permissions to export a graph.

Reply:
Sure, I've figured that the save dialog uses the portal and not the open project, I suppose it's due to not using QFileDialog there?

Reply:
That's right, the dialog for opening a file is different. QFileDialog is not used there.

Reply:
I think `--filesystem=home` would be better here, then. Other than that LGTM.